server/utils/passport/ensure-user.js

```diff
@@ -0,0 +1,31 @@
+import { routes } from '../routes'
```

Comment:
can we change this file name to `ensure-permission` as well?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237#:~:text=%E2%80%A6-,7930718,-Merge%20state